github/pulls.go

```diff
@@ -55,6 +57,7 @@ type PullRequest struct {
 	MaintainerCanModify *bool      `json:"maintainer_can_modify,omitempty"`
 	AuthorAssociation   *string    `json:"author_association,omitempty"`
 	NodeID              *string    `json:"node_id,omitempty"`
+	RequestedReviewers  *[]User    `json:"requested_reviewers,omitempty"` // These are the current requested reviewers of the pull request
```

Comment:
`*[]User` may function equivalently to `[]*User` in practice, however I still recommend `[]*User` for code consistency. See the type of `Assignees` above.

Reply:
changed